modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -554,7 +556,7 @@ func (suite *KeeperTestSuite) TestValidateProposedUpgradeFields() {
 		{
 			name: "change channel version",
 			malleate: func() {
-				proposedUpgrade.Version = "1.0.0"
+				proposedUpgrade.Version = mockUpgradeVersion
```

Comment:
there is `mock.UpgradeVersion` now, can we use that?

Reply:
ah, yes, forgot about that value! i can use it.